context-monitoring/src/test/java/de/atb/context/services/config/TestServiceConfiguration.java

```diff
@@ -21,8 +21,8 @@
 	@Test
 	public void testGetInstance() {
 		String absolutefilePath = new File("").getAbsolutePath();
-		File file = new File(
-				absolutefilePath.concat(File.separator + "resources" + File.separator + "services-config.xml"));
+        File file = new File(
+            absolutefilePath.concat(File.separator + "src" + File.separator + "test" + File.separator + "resources" + File.separator + "services-config.xml"));
 		String filepath = file.getPath();
```

Comment:
`String filePath = Path.of("src", "test", "resources", "services-config.xml").toAbsolutePath().toString()`

Reply:
done